### ISSUE TITLE
There is a problem in using ktimer, could it please be checked?

### ISSUE THREAD

**Issue by rajibando** [kakuilan/ktimer#1]
May please the post no. 6 and 9 of [How to make ktimer play wav files?](https://discuss.kde.org/t/how-to-make-ktimer-play-wav-files/27405/9), and the discussion in LXQt Q&A category, [Does LXQt DE have a bug that convolutes ktimer? Or the other way round? #2665](https://github.com/lxqt/lxqt/discussions/2665) please be perused.

Is the problem because of a bug? Is the bug in LXQt or ktimer?

I use a Debian LXQt system installed from Live ISO 12.6.0. The version for ktimer in my system is 4:22.12.3-1. 

I already have a workaround to the current problem by having the Command line open a terminal and then execute the necessary command: `qterminal -e "bash -c '~/Music/aplay_sound.sh; exec bash'"`, but I don’t like using this trick.

I want to know the reason for the problem and a clear solution.

**Comment by kakuilan** [comment]
I'm sorry, I think you've made a mistake. This ktimer is a private project of mine from a few years ago, it has never been released to the public, it just accepts the url, then periodically visits the url, no other purpose.
Maybe the ktime you're talking about is someone else's project. Look around.

**Comment by rajibando** [comment]
Yes, on hindsight it seems I am mistaken in posting my support request to you, because of the common name `ktimer`. The other `ktimer` is a KDE project. I apologise, and thank you.
